src/app/search/search.resource.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Created by Ilya on 04/11/15.
+ */
+
+'use strict';
+
+var _$resource = new WeakMap();
```

Comment:
I still dont understand: in constructor *set* is immediately followed by *get*, what's the difference between just using *$resource*?